web/packages/shared/components/UnifiedResources/UnifiedResources.tsx

```diff
@@ -113,14 +113,14 @@ export type BulkAction = {
    * over if this prop is supplied
    */
   tooltip?: string;
-  action: (
-    selectedResources: {
-      unifiedResourceId: string;
-      resource: SharedUnifiedResource['resource'];
-    }[]
-  ) => void;
+  action: (selectedResources: SelectedResources) => void;
 };
 
+export type SelectedResources = {
```

Comment:
It looks like this is not actually being used anywhere, though I think it's fine to define it like this instead of cramming everything within `BulkAction` like it was done before.

One thing I'd change, which now just occured to me, is to export `SelectedResource` instead of an array of them and then in the `action` use `SelectedResource[]` instead. I feel like most of the time it's better to export a scalar type like that because making an array out of it is super easy. Doing this the other way around is much harder.